lib/receipt.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+/**
+ * @module striim-sdk
+ */
+
+/**
+ * @class Receipt
+ * A class for modelling a _hubii striim_ payment receipt.
+ */
+class Receipt {
```

Comment:
Should the Receipt class extend Payment? At least intuitively a Payment is very much like a subset of a Receipt in our case.

Reply:
I think it should not extend Payment, because Payment exposes methods that are not applicable to Receipt, for instance "registerPayment". This would likely violate the Liskov Substitution Principle.